mne/preprocessing/maxwell.py

```diff
@@ -642,3 +684,128 @@ def _overlap_projector(data_int, data_res, corr):
     Vh_intersect = Vh_intersect[intersect_mask].T
     V_principal = np.dot(Q_res, Vh_intersect)
     return V_principal
+
+
+def _read_fine_cal(info, fine_cal):
+    """Read sensor locations and calib. coeffs from fine calibration file."""
+
+    # Read new sensor locations
+    with open(fine_cal, 'r') as fid:
+        cal_chans = []
+        for line in fid.readlines():
+            if line[0] == '#' or line[0] == '\n':
+                continue
+
+            # `vals` contains channel number, (x, y, z), x-norm 3-vec, y-norm
+            # 3-vec, z-norm 3-vec, and (1 or 3) imbalance terms
+            vals = np.fromstring(line, sep=' ')
+
+            # Check for correct number of items
+            if len(vals) not in [14, 16]:
+                raise RuntimeError('Error reading fine calibration file')
+
+            ch = int(vals[0].copy())  # Get channel number
+            ch_name = 'MEG' + '%04d' % ch  # Zero-pad names to 4 characters
+            loc = vals[1:13].copy()  # Get orientation information for 'loc'
+
+            # Get orientation information for 'coil_trans'
+            coil_trans = np.zeros((4, 4))
+            coil_trans[3, 3] = 1.
+            coil_trans[0:3, 0:3] = loc[3:].reshape(3, 3).T
+            coil_trans[0:3, 3] = loc[:3]
+
+            calib_coeff = vals[13:].copy()  # Get imbalance/calibration coeff
+            ch_type = channel_type(info, info['ch_names'].index(ch_name))
+
+            cal_chans.append(dict(ch=ch, ch_name=ch_name, loc=loc,
+                                  coil_trans=coil_trans,
+                                  calib_coeff=calib_coeff, ch_type=ch_type))
```

Comment:
These are in device coords, correct? Please add a note. Are MEG channels in `info` typically in device coords, too? Mention if it's the same or different here.